SimTKmath/include/simmath/Optimizer.h

```diff
@@ -309,27 +309,76 @@ class SimTK_SIMMATH_EXPORT OptimizerSystem {
+ * 10^–4 to 10^–3 to have an impact similar to the change from 10^–2 to 10^–1.
+ * In order to avoid the problem that changes of very small values have too less
+ * an impact, an alternative is to choose 10^–1 × (x/10)2 >= 0. In the case where
+ * only a lower bound at zero is necessary, a simple and natural transformation is
+ * x2 × default_x, such that x=1 represents the default (or initial) value and x
+ * remains unbounded during optimization.
+ *
+ * In summary, to map the values [0;10] into [a;b] we have the alternative
+ * transformations a + (b-a) × x/10 or a + (b-a) × (x/10)2 >= a or a × (b/a)x/10 >= 0.
+ *
+ *
  * Advanced options:
  * 
  * The default values for options whose name begins with "stop" are specified
  * at https://github.com/CMA-ES/c-cmaes/blob/master/cmaes_initials.par
  *
  * - <b>popsize</b> (int; default: depends on number of parameters) The
  *   population size (also known as lambda).
- * - <b>init_stepsize</b> (Vector/Real; default: 0.3) Initial stddev; After the encoding of variables,
- *   the initial solution point x0 and the initial standard deviation (step_size) sigma0 must be chosen.
- *   In a practical application, one often wants to start by trying to improve a given solution
- *   locally. In this case we choose a rather small sigma0 (say in [0.001, 0.1], given the x-values
- *   "live" in [0,10]). Thereby we can also check, whether the initial solution is possibly a local
- *   optimum. When a global optimum is sought-after on rugged or multimodal landscapes, sigma0 should be
- *   chosen such that the final desirable location (or at least some of its domain of attraction) is
- *   not far outside of x0 ± 2sigma0 in each coordinate. (Remark that in Rn, if each boundary domain is
- *   in distance sigma, then the boundary corner is sigma*sqrt(n) away, which poses a slight dilemma
- *   for larger n.)
- *   A warning is emitted if this is not set.
+ * - <b>init_stepsize</b> (Vector/Real; default: 0.3) Initial stddev; After the
+ *   encoding of variables, the initial solution point x0 and the initial standard
+ *   deviation (step_size) sigma0 must be chosen. In a practical application, one
+ *   often wants to start by trying to improve a given solution locally. In this
+ *   case we choose a rather small sigma0 (say in [0.001, 0.1], given the x-values
+ *   "live" in [0,10]). Thereby we can also check, whether the initial solution is
+ *   possibly a local optimum. When a global optimum is sought-after on rugged or
+ *   multimodal landscapes, sigma0 should be chosen such that the final desirable
+ *   location (or at least some of its domain of attraction) is not far outside of
+ *   x0 ± 2sigma0 in each coordinate. (Remark that in Rn, if each boundary domain is
+ *   in distance sigma, then the boundary corner is sigma*sqrt(n) away, which poses
+ *   a slight dilemma for larger n.)
+ *
+ *   A warning is emitted if this is not set and the default value is assigned to
+ *   each term of sigma.
```

Comment:
Consider a slightly different wording: "default value is used for each variable."